libs/platform/src/lib/components/search-field/search-field.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="search-input"
```

Comment:
is the class name in fiori3 still `search-input` or should this also be `search-field`?

Reply:
It probable should be `search-field` for naming consistency. I'll make the change.

Reply:
It has been changed.